packages/search/src/types/content.ts

```diff
@@ -61,6 +61,7 @@ export interface IContentSearchFilter {
  * Properties correspond to their Portal API or arcgis-rest-js equivalents
  */
 export interface IContentSearchOptions {
+  site?: string;
```

Comment:
switch to "domain?" "siteDomain"?

Reply:
leave as site, as that works for enterprise and ago, and allows a range of things to be passed in